### DISCUSSION TITLE
Include :root vars

### DISCUSSION THREAD

**Discussion by xbc5** [tailwindlabs/tailwindcss#7280] [Help] [answered]
My custom utility plugins rely on **CSS variables** (theme colors etc).

What's the best way to *implicitly* include these with the plugins? So that I can set them in the config -- e.g:
```css
:root {
  --clr-prim: var(theme("color.prim"));
}
```

Right now I have a :root declaration in a CSS file that's included, but I want it part of the plugins.

I am aware of:
```css
@layer base {
  root: {/*...*/}
}
```
But this requires that I link that stylesheet, which is brittle.

**Comment by aggreggator** [comment, marked as answer]
https://tailwindcss.com/docs/plugins#overview
addBase(), for registering new base styles

**Reply by xbc5** [reply]
Perfect. Thanks!
